cordova-js-src/splashscreen.js

```diff
@@ -19,15 +19,14 @@
  *
 */
 
-var isWp81 = navigator.appVersion.indexOf("Windows Phone 8.1") !== -1;
 var isWp10 = navigator.appVersion.indexOf("Windows Phone 10") !== -1;
-var isPhoneDevice = isWp81 || isWp10;
+var isPhoneDevice = isWp10;
```

Comment:
I would vote to drop `isWp*` and `isPhone*`, we just check if we are dealing with UWP, appx-web, etc.

Reply:
Fell free to do that in a PR, but don't take this as the base as it really was just a PoC to see what happens when I delete some code.